diff --git a/ui/input-linux.c b/ui/input-linux.c
index a7b280b25b..34df9dfa0b 100644
--- a/ui/input-linux.c
+++ b/ui/input-linux.c
@@ -107,7 +107,7 @@ static bool input_linux_check_toggle(InputLinux *il)
     switch (il->grab_toggle) {
     case GRAB_TOGGLE_KEYS_CTRL_CTRL:
         return il->keydown[KEY_LEFTCTRL] &&
-            il->keydown[KEY_RIGHTCTRL];
+            il->keydown[KEY_LEFTALT];
 
     case GRAB_TOGGLE_KEYS_ALT_ALT:
         return il->keydown[KEY_LEFTALT] &&
